osu.Game/Graphics/Containers/ParallaxContainer.cs

```diff
@@ -69,9 +69,11 @@ protected override void Update()
             {
                 Vector2 offset = (input.CurrentState.Mouse == null ? Vector2.Zero : ToLocalSpace(input.CurrentState.Mouse.Position) - DrawSize / 2) * ParallaxAmount;
 
-                double elapsed = MathHelper.Clamp(Clock.ElapsedFrameTime, 0, 1000);
+                const float parallax_duration = 100;
 
-                content.Position = Interpolation.ValueAt(elapsed, content.Position, offset, 0, 1000, Easing.OutQuint);
+                double elapsed = MathHelper.Clamp(Clock.ElapsedFrameTime, 0, parallax_duration);
+
+                content.Position = Interpolation.ValueAt(elapsed, content.Position, offset, 0, parallax_duration, Easing.OutQuint);
                 content.Scale = Interpolation.ValueAt(elapsed, content.Scale, new Vector2(1 + System.Math.Abs(ParallaxAmount)), 0, 1000, Easing.OutQuint);
```

Comment:
Missed this, but looking at the surrounding code again this has the old value too (therefore will stop a tenth of the way in interpolating - which due to the quint easing function isn't *too* bad, but still slightly off), and so does `load()`:
https://github.com/ppy/osu/blob/d400e4a5f6a576b83a355bd34ef405d774382811/osu.Game/Graphics/Containers/ParallaxContainer.cs#L50
Sorry, should have pointed that out before...

Reply:
AH, I actually intentionally left this one, since it only happens on transitions and feels better as a gradual change.